libraries/AP_Baro/AP_Baro_SensorHead.cpp

```diff
@@ -0,0 +1,35 @@
+#include <AP_HAL/AP_HAL.h>
+
+#include "AP_Baro_SensorHead.h"
+#include <AP_BoardConfig/AP_BoardConfig.h>
+
+extern const AP_HAL::HAL& hal;
+
+void BaroMessageHandler::handle(BaroMessage::data_t *data)
+{
+    if (_backend->_sem_baro->take(HAL_SEMAPHORE_BLOCK_FOREVER)) {
+        _backend->_pressure = data->pressure;
+        _backend->_temperature = data->temperature - 273.15f;
```

Comment:
Not really, I had just seen other code using Kelvin. degreesC makes way more sense.